libraries/AP_RangeFinder/RangeFinder.cpp

```diff
@@ -525,6 +526,20 @@ const AP_Param::GroupInfo RangeFinder::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("4_ORIENT", 56, RangeFinder, state[3].orientation, ROTATION_PITCH_270),
 #endif
+
+    AP_SUBGROUPVARPTR(drivers[0], "_",  57, RangeFinder, backend_var_info[0]),
```

Comment:
I understand that this is here because of numbering but that has been "broken" before, the list isn't sequential.